help/include/open-recent-conversations.md

```diff
@@ -0,0 +1,2 @@
+1. Open **Recent conversations** from the left sidebar or by pressing the
+   <kbd>Esc</kbd> key.
```

Comment:
The `Esc` key doesn't always work -- it's depends on a couple of personal settings. Maybe we should just leave that part out.

<img width="455" alt="Screen Shot 2023-03-22 at 11 20 07 PM" src="https://user-images.githubusercontent.com/2090066/227120536-53ce98ed-138b-45a8-a9a2-87df67f2b90d.png">

Reply:
So it should probably just be, "Click **Recent conversations** in the left sidebar." to match https://zulip.com/help/view-your-mentions.